base_rest_abstract_attachment/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "REST API abstract attachment",
+    "summary": "Add an abstract component to manage attachments",
+    "version": "14.0.1.0.0",
+    "website": "https://github.com/OCA/rest-framework",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "base_rest",
+        "base_jsonify",
+        "base_rest_datamodel",
```

Comment:
I would prefer to rely on pydantic in place of datamodel and base_jsonify.

Reply:
done